Userland/Libraries/LibWeb/FileAPI/Blob.h

```diff
@@ -48,14 +48,14 @@ class Blob
 
     virtual JS::Object* create_wrapper(JS::GlobalObject&);
 
+    ByteBuffer byte_buffer() const& { return m_byte_buffer; };
```

Comment:
This is very confusing and I have no idea how that's even valid C++

```suggestion
    ByteBuffer& byte_buffer() { return m_byte_buffer; }
```
?

```suggestion
    ByteBuffer const& byte_buffer() { return m_byte_buffer; }
```
?

```suggestion
    ByteBuffer const& byte_buffer() const { return m_byte_buffer; }
```
?

(in which case just return `ReadonlyBytes` instead)

also no semicolon at the end.

Reply:
FYI it's valid C++ because the `&` on the RHS of the function name means the function is only callable on lvalue reference types. Similarly, `&&` would mean it's only callable on rvalue reference types.

Reply:
Yeah, this was a silly mistake on my part. I have kept it as const reference for now, but can look into a refactoring to ReadonlyBytes in a separate PR